### PR TITLE
Add support of Laravel 5.4

### PR DESCRIPTION
**Pull request by ADiesel82**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "orangehill/iseed",
-    "description": "Generate a new Laravel database seed file based on data from the existing database table.",
+    "name": "adiesel82/iseed",
```
Comment (tihomiro) [inline composer.json]: Can you please leave this line as it is?

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    "description": "Generate a new Laravel 5.4 database seed file based on data from the existing database table.",
```
Comment (tihomiro) [inline composer.json]: Can you also leave this line as it was?

**Comment by tihomiro** [comment]
Thanks for the PR @ADiesel82 ... can you please update the composer.json as per my comments in https://github.com/orangehill/iseed/pull/85/files and I'll merge.

**Comment by tihomiro** [comment]
Already merged  cleaner PR

**Comment by ADiesel82** [comment]
Thanks!

**Pull request closed without merging**